.rubocop.yml

```diff
@@ -117,8 +117,6 @@ Style/Documentation:
     - !ruby/regexp /features\/.*.rb$/
 Style/DoubleNegation:
   Enabled: false
-Style/Encoding:
-  EnforcedStyle: when_needed
```

Comment:
```
Error: obsolete parameter EnforcedStyle (for Style/Encoding) found in .rubocop.yml
Style/Encoding no longer supports styles. The "never" behavior is always assumed.
```

Reply:
I'd like us to have the special encoding header on all our files if possible.

Reply:
@parkr The Ruby Interpreter from v2.0 onwards encodes all files as `UTF-8`..

Reply:
The related Rubocop commit: https://github.com/bbatsov/rubocop/commit/5faf140cd1228749cc6f5b8d5b26543479632082